metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/lineage.py

```diff
@@ -353,34 +357,62 @@ def _parse_exported_bigquery_audit_metadata(
                     f"Unable to parse audit metadata missing {missing_exported_audit} for {audit_metadata}",
                 )
             else:
-                self.report.num_parsed_audit_entires += 1
+                self.report.num_parsed_audit_entries[event.project_id] = (
+                    self.report.num_parsed_audit_entries.get(event.project_id, 0) + 1
+                )
+                self.report.num_total_audit_entries[event.project_id] = (
+                    self.report.num_total_audit_entries.get(event.project_id, 0) + 1
+                )
```

Comment:
Ideally the TopKDict would be a subclass of defaultdict. That'd let this code be simplified

```suggestion
                self.report.num_total_audit_entries[event.project_id] += 1
```